### PR TITLE
fix issue 180 - Rework association names. drop parent_ and child_

### PR DESCRIPTION
**Pull request by elrayle**
Fix for issue https://github.com/projecthydra-labs/hydra-pcdm/issues/180

### CODE REVIEW

**Comment by elrayle** [comment]
The two tests that are failing are failing at master too.  They are not caused by this PR.

file_spec.rb (line 38) - Hydra::PCDM::File technical metadata saves technical metadata
file_spec.rb (line 51) - Hydra::PCDM::File technical metadata does not save server managed properties

Anyone now what is causing these errors?

**Comment by elrayle** [comment]
file_spec.rb (line 38) - marked test pending issue https://github.com/projecthydra-labs/hydra-pcdm/issues/182

file_spec.rb (line 51) - Updates made to ldp changed the error message.  Error message updated and test passes.

**Comment by jcoyne** [comment]
I think you need to adjust the gemspec to require active-fedora >= 9.4.1

File: hydra-pcdm.gemspec [hunk 1]
```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'activefedora-aggregation', '~> 0.4'
+  spec.add_dependency 'activefedora-aggregation', '~> 0.4.1'
```
Comment (jcoyne) [inline hydra-pcdm.gemspec]: Why was this changed?
